scripts/build_release.sh

```diff
@@ -95,6 +95,11 @@ make build
 export BUILD_DEB=1
 scripts/build_packages.sh "${PLATFORM}"
 
+# build docker release package
+pwd
+cd ../docker/release
```

Comment:
Are you sure this is correct?  the previous build_packages.sh script is being called from the root source directory, so this should not require `..` to get there.

Reply:
Fixed this error introduced during while refactoring the docker files.